cairo_programs/field_arithmetic.cairo

```diff
@@ -126,6 +127,105 @@ namespace field_arithmetic {
         }
     }
 
+    // Equivalent of get_square_root but for Uint256
+    func u256_get_square_root{range_check_ptr, bitwise_ptr: BitwiseBuiltin*}(
+        x: Uint256, p: Uint256, generator: Uint256
+    ) -> (success: felt, res: Uint256) {
+        alloc_locals;
+
+        // TODO: Create an equality function within field_arithmetic to avoid overflow bugs
```

Comment:
why this TODO?

Reply:
It's part of the library where the function was extracted, but I agree that it may be better to remove TODOs.